graylog2-server/src/main/java/org/graylog/plugins/views/search/rest/DashboardsResource.java

```diff
@@ -95,6 +96,34 @@ public PageListResponse<ViewSummaryDTO> views(@ApiParam(name = "page") @QueryPar
                                                   @ApiParam(name = "query") @QueryParam("query") String query,
                                                   @ApiParam(name = "filters") @QueryParam("filters") List<String> filters,
                                                   @Context SearchUser searchUser) {
+        return views(page, perPage, sortField, order, query, searchUser::canReadView, filters, searchUser);
+    }
+
+    @GET
+    @ApiOperation("Get a list of all dashboards a user can change")
+    @Path("/canUpdate")
```

Comment:
I would suggest we merge this into the `views` method and just provide a query parameter for the options. IMO it is some kind of a filter for that endpoint.

Reply:
will do